extensions/wikia/AdEngine/js/video/ooyalaAdSetProvider.js

```diff
@@ -0,0 +1,69 @@
+/*global define*/
+define('ext.wikia.adEngine.video.ooyalaAdSetProvider', [
+	require.optional('ext.wikia.adEngine.adContext'),
```

Comment:
It could be if other platform has a problem with loading ads / ad context, we will return empty array in that case without throwing an error, content video will play anyway. Buuuuuttt... maybe there is better solution for that